### DISCUSSION TITLE
What are the green ++ and red -- symbols?

### DISCUSSION THREAD

**Discussion by NorielSylvire** [community/community#22315] [New to GitHub]
<p>Well, I’m a high school student so I’m not an actual programmer, just as a hobbie. But I like it and want to make my code free and open source (and I also want to start a project with a friend) so that’s why I started using GitHub. But I don’t use it very often, mostly becasue I have to study a lot, and I don’t know how it works.</p>
<p>So I don’t know what these green ++ and red – symbols besides the name of each contributor are. You can see them after you click on “Contributors”. Are they like the likes and dislikes of YouTube?</p>
<p>If so, wow, I got a couple of thousand and it kinda scares me to know that so many people have seen my code. Which is actually a good thing, cause that means that I will work hard to do it as good as I can so that people can enjoy and not hate my work.</p>
<p>If not, what are these crosses and lines?</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/fl4v1u5">@fl4v1u5</a>,</p>
<p>There are a lot of places in a github repository where you see the ++ and --. Most of the time indicates <strong>how many lines you add or remove</strong>. For example:</p>
<p><img src="https://user-images.githubusercontent.com/12409541/181070160-6a537d2e-9b7e-493f-afdc-4f16af00a7d6.png" alt="ex.png" data-base62-sha1="uM58UVourZiVWNTQuMDPrUysyG" width="468" height="210"></p>
<p>You have added from the initial commit of the repository to now 1998 new lines of code and delete/replace 84 lines of code. Most of the time when you work in a project that contains images, graphics, video or media files in general you have a big amount of ++ because github counts the lines of the binary file (I think).</p>
<p>If you want to see how many people watch your repository you need to go to the <strong>traffic</strong> on the <strong>insights</strong> tab:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/4f88cbbc7fb8a078aa801dd81a8467461de5f671.png" data-download-href="/uploads/short-url/blANQeFS1DxtGr6tymuZ1keXvfb.png?dl=1" title="here.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181070161-4af3edea-9124-4f3d-aa9d-4d3c2b177220.png" alt="here.png" data-base62-sha1="blANQeFS1DxtGr6tymuZ1keXvfb" width="497" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/4f88cbbc7fb8a078aa801dd81a8467461de5f671_2_497x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/4/4f88cbbc7fb8a078aa801dd81a8467461de5f671_2_745x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/4/4f88cbbc7fb8a078aa801dd81a8467461de5f671.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/4f88cbbc7fb8a078aa801dd81a8467461de5f671_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">here.png</span><span class="informations">916×920 116 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The red one is the diagram of how many times the repository has been cloned.<br>
The green one is the diagram of how many people have visited your repository.<br>
The last blue is a summary of where the repository was discovered and which files are watched.</p>
<p>Hope this helps :slightly_smiling_face:  </p>
<p>-Gabriele-</p>
<blockquote>
<p><strong>Mark helpful posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great content!</strong></p>
</blockquote>

**Comment by NorielSylvire** [comment]
<p>Woah, I had no idea. It does help a lot! Thank you very much, Grabriele!</p>

**Comment by vyanky1227** [comment]
The green "++" symbol in GitHub typically represents additions or lines of code that have been added or modified in a pull request or a commit. It indicates an increase in the number of lines or changes made.

Conversely, the red "--" symbol represents deletions or lines of code that have been removed or modified in a pull request or a commit. It indicates a decrease in the number of lines or changes made.
